src/server.h

```diff
@@ -3894,14 +3894,30 @@ void incrementMvccTstamp();
 
 #if __GNUC__ >= 7 && !defined(NO_DEPRECATE_FREE)
  [[deprecated]]
+#ifdef ALPINE
+void *calloc(size_t count, size_t size);
```

Comment:
Alping is just not standards compliant?  Maybe your testing on a tool old version?

Reply:
tested on latest alpine docker image